lib/stanford-corenlp-full-2015-12-09/patterns/example.properties

```diff
@@ -0,0 +1,297 @@
+#### NOTE: for all flags and their description, see the javadoc. Important parameters (in our experience) that you should tune for your dataset are marked with ***. Pay special attention to flags like targetAllowedNERs. Use batchProcesssSents and Lucene based options if you want low memory (but slower) runs.
```

Comment:
This should probably not be here either.  Neither should the source jars mentioned above.